src/libraries/Microsoft.Win32.Registry/src/Microsoft.Win32.Registry.csproj

```diff
@@ -2,7 +2,7 @@
   <PropertyGroup>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <DefineConstants>$(DefineConstants);REGISTRY_ASSEMBLY</DefineConstants>
-    <TargetFrameworks>$(NetCoreAppCurrent)-Windows_NT;$(NetCoreAppCurrent)-Unix;$(NetFrameworkCurrent)-Windows_NT;netstandard2.0-Windows_NT;netstandard2.0-Unix;netstandard2.0;net461-Windows_NT</TargetFrameworks>
+    <TargetFrameworks>$(NetCoreAppCurrent)-Windows_NT;$(NetCoreAppCurrent);$(NetFrameworkCurrent)-Windows_NT;netstandard2.0-Windows_NT;netstandard2.0;net461-Windows_NT</TargetFrameworks>
```

Comment:
@ericstj I'm curious, why are we even building Unix and Browser configurations (now collapsed as RIDless) but later excluding them from the package?

Reply:
I guess it is because it is inbox?

https://github.com/dotnet/runtime/blob/ccfe841671fc3dca5aa800220c350ba570760c01/src/libraries/Microsoft.Win32.Registry/Directory.Build.props#L5

Reply:
@Anipik we aren't testing the netstandard configuration (and I don't exactly now what it is for?). Is that intentional? https://github.com/dotnet/runtime/blob/ccfe841671fc3dca5aa800220c350ba570760c01/src/libraries/Microsoft.Win32.Registry/tests/Microsoft.Win32.Registry.Tests.csproj#L4

Reply:
the netstandard configuration is what goes in the package for compatible runtimes of netstandard2.0.